sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -382,6 +396,58 @@ def _convert_span_to_envelope(span: Span) -> TelemetryItem:
         data.properties["_MS.links"] = json.dumps(links)
     return envelope
 
+def _convert_span_events_to_envelopes(span: Span) -> Sequence[TelemetryItem]:
+    envelopes = []
+    for event in span.events:
+        envelope = TelemetryItem(
+            name="",
+            instrumentation_key="",
+            tags=dict(_utils.azure_monitor_context),
+            time=ns_to_iso_str(event.timestamp),
+        )
+        # pylint: disable=protected-access
+        envelope.tags.update(_utils._populate_part_a_fields(span.resource))
+
+        envelope.tags["ai.operation.id"] = "{:032x}".format(span.context.trace_id)
+        if span.parent and span.parent.span_id:
+            envelope.tags["ai.operation.parentId"] = "{:016x}".format(
+                span.parent.span_id
+            )
+        properties = {}
```

Comment:
Looks like all new code until here is the same for spanToEnvelope method, maybe create a shared method for it?

Reply:
Abstracted out `TelemetryItem` creation. Will leave in `operation_id` as it is because the logic is slightly different.